docs/user/class-chests.rst

```diff
@@ -2,105 +2,123 @@
 Class chests
 ############
 
-**On this page:** \* `About Class Chests <#about-class-chests>`__ \*
-`Linked Class Chests <#linked-class-chests>`__
-
-About Class Chests
-------------------
-
-If some of your favorite items aren't supported by MobArena's internal
-[[item parser\|Item and Reward Syntax]], or if you just want to be able
-to configure your class items from in-game, the **class chests** may be
-what you're looking for!
-
---------------
-
-**Note: The Class Chests will only work for arenas with
-``use-class-chests: true``, and the classes *MUST* exist in the
-config-file for MobArena to recognize them, however the items and armor
-lists can be empty.** \* \* \*
-
-The idea behind the class chests is to simply place some **chests below
-the corresponding class signs** in the lobby, and fill them with
-whatever items you want the given class to have. When the players
-activate the class signs, the **contents of the chests are copied to the
-player inventory**. This suggests a type of "control room" setup, where
-an admin-only access room below the lobby contains the chests, allowing
-admins to warp down there and change the contents of the chests.
-
-.. figure:: img/1.png
-   :alt: Lobby and Control Room
-
-   Lobby and Control Room
-
-For easier access and modification of the class chests, omitting the
-control room from the arena or lobby region may prove useful. Otherwise,
-arenas may have to be temporarily disabled or put into edit mode to
-allow warping to and changing the contents of the chests.
```

Comment:
Because of MobArena's overprotective nature, this paragraph about placing the control outside of the arena/lobby region may be more important than one would think. People seem to struggle quite a bit with edit mode. Positioning the control room outside of the region allows people to change the chest contents without having to enter edit mode first. It should still be admin-accessible only to prevent tampering.

Reply:
Ohh, I understand this now. I didn't understand it on the first read, but this explanation makes sense. I'll write this back in.